template/apax.yml

```diff
@@ -24,8 +24,6 @@ apaxVersion: 3.2.1
 devDependencies:
   '@ax/sdk': 2405.0.0
   "@simatic-ax/snippetscollection": 1.0.0
-dependencies:
-  "@ax/system": 7.0.17 # this is an example lib-dependency for your code, pls. adjust
```

Comment:
I have added that because otherwise (no dependencies section) by adding new dependencies they will always be generated at the buttom of the apax.yml. Having them already next to the devdependencies  (by default) makes a better look an feel in the end if noone deosn't care about the pattern.

Reply:
For many reasons it's acceptable that the dependency section is at the end of the file. Or is there any reasonable pain? For my opinion it's more annoying to delete unused dependencies afterwards (especially including the complete system library which blows up your project unnecessarily).

Reply:
There is no reasonal pain having it or leaving it as it is ... Iam fine of deleting it again.
> MR can be merged :)